scout/server/blueprints/genes/views.py

```diff
@@ -50,6 +50,11 @@ def gene(hgnc_id=None, hgnc_symbol=None):
 def api_genes():
     """Return JSON data about genes."""
     query = request.args.get("query")
-    build = request.args.get("build", "all")
-    json_out = controllers.genes_to_json(store, query, build)
-    return jsonify(json_out)
+    if query is None:
+        return jsonify({"code": 400, "message": "missing 'query' param in request"})
+    build = request.args.get("build", "37")
+    try:
+        json_out = controllers.genes_to_json(store, query, build)
+        return jsonify(json_out)
+    except Exception as ex:
```

Comment:
I seem to recall some design pattern not liking excepting general exceptions. I suppose the idea is you only Except what your really expect to see and understand, not other random stuff that you didn't really think about yet. But as it is directly just going to an error forwarded to the API user, I don't quite see why it would be bad here. 👍

Reply:
The thing is we don't know what to expect here, hopefully no errors. I was reasoning that it's better to have any error handled than no errors handled

Reply:
I think it is fine - it kind of moves the error checking one step away, but at the same time, we mostly expect to be the ones using that api on the other end from other apps, so all fine.